src/main/java/com/truelayer/java/payments/entities/paymentdetail/Retry.java

```diff
@@ -0,0 +1,13 @@
+package com.truelayer.java.payments.entities.paymentdetail;
+
+import java.util.List;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class Retry extends AuthorizationFlowAction {
+    Type type = Type.RETRY;
+
+    List<String> retryOptions;
```

Comment:
Do we want to model this as a list of enum? We usually follow this approach (and I've pinned for rework in our gap analysis where we don't). Example [here](https://github.com/TrueLayer/truelayer-java/blob/b5ba51ba559ff0d8b14776461b5dc320a18e73e7/src/main/java/com/truelayer/java/entities/ProviderFilter.java#L21)